src/main/java/me/totalfreedom/totalfreedommod/command/Command_mute.java

```diff
@@ -130,17 +130,19 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         {
             playerdata.setMuted(true);
             player.sendTitle(ChatColor.RED + "You've been muted.", ChatColor.YELLOW + "Be sure to follow the rules!", 20, 100, 60);
-            if (reason != null)
-            {
-                msg(player, ChatColor.RED + "Reason: " + ChatColor.YELLOW + reason);
-            }
+            
             if (quiet)
             {
                 msg("Muted " + player.getName() + " quietly");
-                return true;
+                return true;  // doesn't announce reason
```

Comment:
the code you moved didn't ever announce the reason, wtf?

Reply:
Note the "quietly". Why would it announce the reason if it was intended for quiet announcement?